xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/displayer_page.vm

```diff
@@ -1 +1,23 @@
-#template('displayer_dblist.vm')
+#if ($type == 'edit' || $type == 'search')
+  #if ($field.getProperty('displayType').value == 'input' && $field.getProperty('picker').value == 1)
+    #xpropertySuggestInputDisplayer($field $prefix $name $value)
+  #else
+    $doc.displayEdit($field, $prefix, $object)
+  #end
+#elseif ($type == 'view')
+  #set ($value = $doc.displayView($field, $prefix, $object))
+  #set ($pageDoc = $xwiki.getDocument($value))
+  #if ("$!pageDoc" == "" || $pageDoc.isNew())
+    $value
```

Comment:
You don't need to get the document to obtain the URL. The value is a document reference and you can get the URL with that so you can show a link to the missing document (using the proper CSS class to show the question mark, see what happens when you create a link to a missing wiki page from wiki syntax).

Reply:
@mflorea Seems like the `$value` variable is either a String or a List.

Reply:
Yes, you need to handle multiple selection. See the user displayer.